src/mail.sh

```diff
@@ -1119,6 +1126,8 @@ function parse_mail_options()
         for i in "$@"; do
           if [[ "${i}" =~ " " ]]; then
             pass_option_to_send_email+=" ${i@Q}"
+          elif [[ -f "$i" || -d "$i" ]]; then
+            options_values['PATCHES_SOURCE']="$i"
```

Comment:
The interface is not clear to me. You must update the man page by adding how this new option looks like.